modules/common-attributes.adoc

```diff
@@ -51,3 +51,6 @@ endif::[]
 :smallpluralporv: processes
 :capsporv: Process
 :capspluralporv: Processes
+// Following variables are required for publishing using PV2
+:product-title: OpenShift Container Platform
+:product-version: 3.65
```

Comment:
@vikram-redhat Can you confirm these changes. The preview looks good and substitutes the instances properly.